learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Analytics
+description: Meilisearch Cloud offers in-depth search analytics to help you understand how users search in your application.
+sidebarDepth: 3
+---
+
+# Analytics
+
+Enable Meilisearch Cloud analytics to gain insight on what your users are searching for, how often they find it, and how much they have to wait between typing a query and receiving results.
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
+
+## Activating analytics
+
+### Enable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and click on the "Enable analytics" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics", and a short explanation of the feature.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics" section will update when the feature is enabled.
+
+### Update URLs in your application
+
+Enabling analytics generates a new URL for your project. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
+
+Update your application so all API requests point to the new URL:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+Your old project URL will remain functional, but API calls using the old URL will not be reflected in the analytics interface.
+
+<Capsule intent="note" title="Analytics and latency">
+To capture and process search analytics data, Meilisearch Cloud must route API requests through a proxy. This might increase latency in some applications.
```

Comment:
This currently gives me more of a "warning, you might not want to do that" feel, where in reality it's not that big a deal for most users. Do you think we could be reassuring/informative here by giving numbers?

afaik the latency only increase by 10 to 25 ms (@brunoocasali @qdequele please correct me if I'm wrong)

Reply:
This does feel like the right thing to do to me @Strift. Because we usually mention type-as-you-search, search responses under 50ms and other topics like this one. So it may not seem that much (for you and me), but since we have that kind of statement, we need to be clear in my POV that this has some drawbacks (but also, a lot of benefits).

Reply:
To clarify, I'm not against giving a warning. But I'd love it if we could make it a bit more reassuring by giving estimated latency numbers. This way, the customer has the full picture.